src/whylogs/core/metrics/confusion_matrix.py

```diff
@@ -0,0 +1,148 @@
+from typing import List, Union
+
+import numpy as np
+from sklearn.utils.multiclass import type_of_target
+
+from whylogs.proto import ScoreMatrixMessage
+from whylogs.core.statistics import NumberTracker
+
+
+SUPPORTED_TYPES = ("binary", "multiclass")
+
+
+class ConfusionMatrix:
+
+    """
+    Confusion Matrix Class to hold labels and matrix data
+
+    Attributes:
+        confusion_matrix (nd.array): Description
```

Comment:
I call this `scoresMatrix` in Java FYI since it's more than a confusion matrix.

Also missing description

Reply:
👍 agree will change